### ISSUE TITLE
What's the status of the "multithreading" branch?

### ISSUE THREAD

**Issue by tsao-chi** [tr7zw/YAPFA#6]
Is it deprecated?

**Comment by tr7zw** [comment]
For now, it's on hold(outdated). The branch *should* work, but I didn't have time to do some deeper testing. The plan is to port my 1.8 paper forks multithreading to 1.15. For now, it ticks all worlds in parallel, but in the future maybe more could also be ticked at the same time.

**Comment by tr7zw** [comment]
Ok I renamed the branch to be just an archive. For now this will just remain there.
